src/richie/plugins/glimpse/models.py

```diff
@@ -0,0 +1,86 @@
+"""
+Glimpse plugin models
+"""
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.text import Truncator
+from django.utils.translation import ugettext_lazy as _
+
+from cms.models.fields import PageField
+from cms.models.pluginmodel import CMSPlugin
+from filer.fields.image import FilerImageField
+
+from .defaults import GLIMPSE_VARIANTS
+
+
+@python_2_unicode_compatible
+class Glimpse(CMSPlugin):
+    """
+    Glimpse plugin model
+    """
+
+    title = models.CharField(max_length=255, blank=True, null=True)
+    variant = models.CharField(
+        _("Variant"),
+        max_length=50,
+        choices=GLIMPSE_VARIANTS,
+        default=GLIMPSE_VARIANTS[0][0],
+        blank=True,
+        null=True,
+        help_text=_("Form factor variant"),
+    )
+    image = FilerImageField(
+        related_name="image",
+        verbose_name=_("image"),
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+    content = models.TextField(_("Content"), blank=False, default="")
+    link_url = models.URLField(
+        verbose_name=_("External URL"),
+        blank=True,
+        null=True,
+        max_length=255,
+        help_text=_("Make the glimpse as a link with an external URL."),
+    )
+    link_page = PageField(
+        verbose_name=_("Internal URL"),
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text=_("Make the glimpse as a link with an internal (page) URL."),
+    )
+
+    def __str__(self):
+        return Truncator(self.title).words(6, truncate="...")
+
+    def get_link(self):
+        """
+        Return attached link if any.
+
+        In case of "link_url" and "link_page" both not empty, "link_url" is
+        always used.
+        """
+        link = False
```

Comment:
Returning a boolean for a link seems broken. Why not return "None"?
```suggestion
        link = None
```